src/js/pages/organizer/Notifications.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+
+import NotificationStore from 'stores/NotificationStore';
+
+const moment = require('moment');
+
+
+export default class Notifications extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      notifications: [],
+    };
+
+    NotificationStore.getAll();
+  }
+
+  componentWillMount() {
+    NotificationStore.on('received', this.loadNotifications);
+    NotificationStore.on('created', this.loadNotifications);
+    NotificationStore.on('addNotification', this.loadNotifications);
+    NotificationStore.on('deleteNotification', this.loadNotifications);
+    NotificationStore.on('updateNotification', this.loadNotifications);
+    NotificationStore.on('error', this.showError);
+  }
+
+  componentWillUnmount() {
+    NotificationStore.removeListener('received', this.loadNotifications);
+    NotificationStore.removeListener('created', this.loadNotifications);
+    NotificationStore.removeListener('addNotification', this.loadNotifications);
+    NotificationStore.removeListener('deleteNotification', this.loadNotifications);
+    NotificationStore.removeListener('updateNotification', this.loadNotifications);
+    NotificationStore.removeListener('error', this.showError);
+  }
+
+  loadNotifications = () => {
+    this.setState({ notifications: NotificationStore.notifications });
+  }
+
+  showError = () => {
+    console.error(NotificationStore.error);
+  }
+
+  updateNotification = (row) => {
+    const data = row;
+    data.expirytime = moment(row.expirytime, 'YYYY-MM-DDTHH:mm').valueOf();
+    NotificationStore.update(data);
+  }
+
+  addNotification = (row) => {
+    NotificationStore.add(row);
+  }
+
+  deleteNotification = (row) => {
+    row.forEach(id => NotificationStore.delete(id));
+  }
+
+  formatExpiry = (cell) => {
+    const format = 'MMM Do hh:mm a';
+    return moment(cell).format(format);
+  }
+
+  insertModal = (columns, validateState, ignoreEditable) => {
+    return (<InsertModalBody columns={columns} validateState={validateState} ignoreEditable={ignoreEditable} />);
```

Comment:
The calendar table doesnt work so I'm going to leave this as is at look at fixing the calendar table to match it.